index_countdown.html

```diff
@@ -20,100 +20,135 @@
 </head>
 
 <body class="home static">
-
-	<!-- ## SITE NAV ## -->
-	<aside id="qcSiteNav">
-		<div class="box">
-			<h4>NAvigation</h4>
-			<ul id="qcPriNav">
-				<li class="active"><a href="index.html">Home</a></li>
-				<li><a href="#">Event</a>
-					<!-- ## SUBMENU ## -->
-					<ul>
-						<li><a href="schedule.html">Schedule</a></li>
-						<li><a href="speakers.html">Speakers</a></li>
-						<li><a href="venue.html">Venue</a></li>
-						<li><a href="gallery.html">Gallery</a></li>
-						<li><a href="ticket.html">Tickets</a></li>
-						<li><a href="faq.html">FAQ's</a></li>
-					</ul>
-				</li>
-				<li><a href="about.html">About</a></li>
-				<li><a href="contact.html">Contact</a></li>
-				<li><a href="#">Login / Signup</a></li>
-			</ul>
-			<div id="qcCloseNav"></div>
-		</div>
-	</aside>
-	<!-- ## SITE NAV END ## -->
-
-	<!-- ## PRIMARY HEADER ## -->
-	<header id="qcHeader">
-		<div class="qcContainer clearfix">
-			<div id="qcNavLogo" class="col-6 col">
-				<a href="http://www.writespeakcode.com" target="_blank">
-					<img src="./assets/img/WSClogos_stacked_coral-blush-gold_screen.png" width="200" alt="LOGO" />
-				</a>
+	<div class="countdown">
+		<!-- ## SITE NAV ## -->
+		<aside id="qcSiteNav">
+			<div class="box">
+				<h4>Navigation</h4>
+				<ul id="qcPriNav">
+					<li class="active"><a href="index.html">Home</a></li>
```

Comment:
Can you rename the `index_countdown.html` file to `index.html`? That should fix this link, and reflect the fact that we have only one index page :)